src/lib/components/atoms/SpokenText.svelte

```diff
@@ -23,13 +23,11 @@
 
 <style>
 	button {
-		line-height: 1;
 		border: none;
-		cursor: pointer;
 		background: none;
-		height: var(--space-m);
-		width: var(--space-m);
-		padding: 0.4em 0 0;
+		height: var(--space-l);
+		width: var(--space-l);
+		padding: 0.4rem 0 0;
```

Comment:
This is static, while the space variables are dynamic to the screen width so this might break when scaling down